src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -4185,6 +4185,66 @@ public void WriteNumberValueDecimal(bool formatted, bool skipValidation)
             AssertContents(expectedStr, output);
         }
 
+        [Theory]
+        [InlineData(true, true, "message")]
+        [InlineData(true, false, "message")]
+        [InlineData(false, true, "message")]
+        [InlineData(false, false, "message")]
+        [InlineData(true, true, "mess><age")]
+        [InlineData(true, false, "mess><age")]
+        [InlineData(false, true, "mess><age")]
+        [InlineData(false, false, "mess><age")]
+        [InlineData(true, true, ">><++>>>\">>\\>>&>>>\u6f22\u5B57>>>")] // \u6f22 = 汉, \u5B57 = 字
+        [InlineData(true, false, ">><++>>>\">>\\>>&>>>\u6f22\u5B57>>>")]
+        [InlineData(false, true, ">><++>>>\">>\\>>&>>>\u6f22\u5B57>>>")]
+        [InlineData(false, false, ">><++>>>\">>\\>>&>>>\u6f22\u5B57>>>")]
+        [InlineData(true, true, "mess\r\nage\u0008\u0001!")]
+        [InlineData(true, false, "mess\r\nage\u0008\u0001!")]
+        [InlineData(false, true, "mess\r\nage\u0008\u0001!")]
+        [InlineData(false, false, "mess\r\nage\u0008\u0001!")]
+        public void WriteStringsWithRelaxedEscaping(bool formatted, bool skipValidation, string keyString)
+        {
+            string expectedStr = GetExpectedString_RelaxedEscaping(prettyPrint: formatted, keyString);
+
+            var options = new JsonWriterOptions { Indented = formatted, SkipValidation = skipValidation, Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping };
+
+            ReadOnlySpan<char> keyUtf16 = keyString.AsSpan();
+            ReadOnlySpan<byte> keyUtf8 = Encoding.UTF8.GetBytes(keyString);
+
+            for (int i = 0; i < 3; i++)
+            {
+                var output = new ArrayBufferWriter<byte>(1024);
+                var jsonUtf8 = new Utf8JsonWriter(output, options);
```

Comment:
Utf8JsonWriter's IDisposable, so it should be in a using :smile:

Reply:
Fixed it everywhere within the test project (lots of places didn't dispose it).

Reply:
Actually, I'd recommend undoing your commit that added it everywhere.  There's so much noise in this change now it's hard to see what you're semantically changing.

It's a good followup change, though (and not needed for 3.0 porting)